src/nvim/po/de.po

```diff
@@ -153,476 +396,1466 @@ msgstr ""
 msgid "Scanning tags."
 msgstr "Durchsuchen von Tags."
 
-#: ../edit.c:4418
+#: ../edit.c:4639
 msgid " Adding"
 msgstr " F�ge hinzu"
 
 #. showmode might reset the internal line pointers, so it must
 #. * be called before line = ml_get(), or when this address is no
 #. * longer needed.  -- Acevedo.
 #.
-#: ../edit.c:4461
+#: ../edit.c:4682
 msgid "-- Searching..."
 msgstr "-- Suche..."
 
-#: ../edit.c:4517
+#: ../edit.c:4741
 msgid "Back at original"
 msgstr "Zur�ck am Ursprung"
 
-#: ../edit.c:4520
+#: ../edit.c:4744
 msgid "Word from other line"
 msgstr "Wort aus anderer Zeile"
 
-#: ../edit.c:4523
+#: ../edit.c:4747
 msgid "The only match"
 msgstr "Einziger Treffer"
 
-#: ../edit.c:4579
+#: ../edit.c:4803
 #, c-format
 msgid "match %d of %d"
 msgstr "Treffer %d von %d"
 
-#: ../edit.c:4583
+#: ../edit.c:4807
 #, c-format
 msgid "match %d"
 msgstr "Treffer %d"
 
-#: ../ex_cmds.c:119
-#, c-format
-msgid "<%s>%s%s  %d,  Hex %02x,  Octal %03o"
-msgstr "<%s>%s%s  %d,  Hex %02x,  Oktal %03o"
+#: ../eval.c:152
+msgid "E18: Unexpected characters in :let"
+msgstr "E18: Unerwartete Zeichen in :let"
 
-#: ../ex_cmds.c:142
+#: ../eval.c:153
 #, c-format
-msgid "> %d, Hex %04x, Octal %o"
-msgstr "> %d, Hex %04x, Oktal %o"
+msgid "E684: list index out of range: %<PRId64>"
+msgstr "E684: List-Index au�erhalb des zul�ssigen Bereichs: %<PRId64>"
 
-#: ../ex_cmds.c:143
+#: ../eval.c:154
 #, c-format
-msgid "> %d, Hex %08x, Octal %o"
-msgstr "> %d, Hex %08x, Oktal %o"
-
-#: ../ex_cmds.c:679
-msgid "E134: Move lines into themselves"
-msgstr "E134: Verschiebe Zeilen in sich selbst"
+msgid "E121: Undefined variable: %s"
+msgstr "E121: Nicht definierte Variable: %s"
 
-#: ../ex_cmds.c:742
-msgid "1 line moved"
-msgstr "eine Zeile verschoben"
+#: ../eval.c:155
+msgid "E111: Missing ']'"
+msgstr "E111: Fehlende ']'"
 
-#: ../ex_cmds.c:744
+#: ../eval.c:156
 #, c-format
-msgid "%<PRId64> lines moved"
-msgstr "%<PRId64> Zeilen verschoben"
+msgid "E686: Argument of %s must be a List"
+msgstr "E686: Argument von %s muss vom Typ List sein"
 
-#: ../ex_cmds.c:1170
+#: ../eval.c:158
 #, c-format
-msgid "%<PRId64> lines filtered"
-msgstr "%<PRId64> Zeilen gefiltert"
+msgid "E712: Argument of %s must be a List or Dictionary"
+msgstr "E712: Argument von %s muss vom Typ List oder Dictionary sein"
 
-#: ../ex_cmds.c:1189
-msgid "E135: *Filter* Autocommands must not change current buffer"
-msgstr "E135: *Filter*-Autokommandos d�rfen den aktuellen Puffer nicht �ndern"
+#: ../eval.c:159
+msgid "E713: Cannot use empty key for Dictionary"
+msgstr "E713: Schl�ssel f�r Dictionary darf nicht leer sein"
 
-#: ../ex_cmds.c:1239
-msgid "[No write since last change]\n"
-msgstr "[Kein Schreiben seit der letzten �nderung]\n"
+#: ../eval.c:160
+msgid "E714: List required"
+msgstr "E714: List ben�tigt"
 
-#: ../ex_cmds.c:1418
-#, c-format
-msgid "%sviminfo: %s in line: "
-msgstr "%sviminfo: %s in Zeile: "
+#: ../eval.c:161
+msgid "E715: Dictionary required"
+msgstr "E715: Dictionary ben�tigt"
```

Comment:
With 'Dictionary' it's better than with 'List', as it's not close to a German word, but if we go for 'List-Objekt' or '&#96;List&#96;'. we should go for 'Dictionary-Objekt' or '&#96;Dictionary&#96;', too.